language/move-prover/doc/user/spec-lang.md

```diff
@@ -188,8 +197,8 @@ one can use the notation `::len(v)`.
 
 ## Partial Semantics
 
-In MSL, expressions have a partial semantics. This is in contrast to Move, where they have a total semantics, since they
-either deliver a value or abort.
+In MSL, expressions have a partial semantics. This is in contrast to Move program, where expression have a total
```

Comment:
I suggest: "...contrast to Move program expressions, which have..."

Reply:
Done